projects/onto-search/src/lib/onto-search-facet/range-facet/range-slider/range-slider.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="range-slider-wrapper">
   <div class="range-slider">
-    <div class="min-max-slider" #minMaxSlider>
+    <div class="slider" #minMaxSlider>
```

Comment:
Actually I didn't see the class name. I meant the `#minMaxSlider`

Reply:
done